src/FluentValidation/Validators/CustomValidator.cs

```diff
@@ -34,7 +33,7 @@ public class CustomValidator<T> : PropertyValidator {
 			_isAsync = true;
 			_asyncAction = asyncAction;
 			//TODO: For FV 9, throw an exception by default if async validator is being executed synchronously.
-			_action = (x, ctx) => Task.Run(() => _asyncAction(x, ctx, new CancellationToken())).GetAwaiter().GetResult();
+            _action = (x, ctx) => Task.Run(() => _asyncAction(x, ctx, new CancellationToken())).GetAwaiter().GetResult();

```

Comment:
Can you revert the indentation change too please?

Reply:
done